sbi/inference/posteriors/mcmc_posterior.py

```diff
@@ -94,9 +99,18 @@ def __init__(
         self.warmup_steps = warmup_steps
         self.num_chains = num_chains
         self.init_strategy = init_strategy
-        self.init_strategy_num_candidates = init_strategy_num_candidates
+        self.init_strategy_parameters = init_strategy_parameters
         self.num_workers = num_workers
 
+        if init_strategy_num_candidates is not None:
+            warn(
+                """Passing `init_strategy_num_candidates` is deprecated as of sbi
+                v0.19.0. Pass a dictionary containing init strategy kwargs instead."""
```

Comment:
I would suggest making this more explicit:

Passing `init_strategy_num_candidates` is deprecated as of sbi v0.19.0. Instead, use`init_strategy_parameters={"num_candidate_samples": 1000}`